massdriver-application-azure-app-service/_variables.tf

```diff
@@ -15,6 +15,8 @@ variable "image" {
     registry = string
     name     = string
     tag      = string
+    username = optional(string)
+    password = optional(string)
```

Comment:
These are new fields added into the Terraform, which are tested for DockerHub private repos. This is now an optional field for those who want to pull images from their private repo.